### PR TITLE
Finish glimpse -> gimp migration

### PR DESCRIPTION
**Pull request by bkarnow**
Last part of migration - internal text and quest name changes
Text changes included otherwise they'll break master.

https://phabricator.endlessm.com/T30805

### CODE REVIEW

File: eosclubhouse/quests/hack2/gimpphoto1.py [hunk 1]
```diff
@@ -21,23 +21,24 @@
...
 
 
-class GlimpsePhoto1(Quest):
+class GIMPPhoto1(Quest):
 
     __app_id__ = 'org.gimp.GIMP'
     __app_common_install_name__ = 'GIMP'
     __tags__ = ['pathway:art', 'difficulty:easy', 'since:1.8']
     __pathway_order__ = 640
+    __app_common_install_name__ = "GIMP"
```
Comment (manuq) [inline eosclubhouse/quests/hack2/gimpphoto1.py:30]: This line is repeated, see 3 lines above.

File: eosclubhouse/quests/hack2/gimpphoto2.py [hunk 2]
```diff
@@ -21,15 +21,16 @@
...
 
 
-class GlimpsePhoto2(Quest):
+class GIMPPhoto2(Quest):
 
     __app_id__ = 'org.gimp.GIMP'
     __app_common_install_name__ = 'GIMP'
     __tags__ = ['pathway:art', 'difficulty:easy', 'since:1.8']
     __pathway_order__ = 641
+    __app_common_install_name__ = "GIMP"
```
Comment (manuq) [inline eosclubhouse/quests/hack2/gimpphoto2.py:30]: This line is repeated, see 3 lines above.

File: eosclubhouse/quests/hack2/gimpphoto3.py [hunk 3]
```diff
@@ -21,23 +21,24 @@
...
 
 
-class GlimpsePhoto3(Quest):
+class GIMPPhoto3(Quest):
 
     __app_id__ = 'org.gimp.GIMP'
     __app_common_install_name__ = 'GIMP'
     __tags__ = ['pathway:art', 'difficulty:easy', 'since:1.8']
     __pathway_order__ = 642
+    __app_common_install_name__ = "GIMP"
```
Comment (manuq) [inline eosclubhouse/quests/hack2/gimpphoto3.py:30]: This line is repeated, see 3 lines above.

File: eosclubhouse/quests/hack2/newcursor.py [hunk 4]
```diff
@@ -26,16 +26,13 @@ class NewCursor(Quest):
     __app_id__ = 'org.gimp.GIMP'
     __tags__ = ['pathway:operating system', 'difficulty:hard', 'since:1.6']
     __pathway_order__ = 420
+    __app_common_install_name__ = "GIMP"
```
Comment (manuq) [inline eosclubhouse/quests/hack2/newcursor.py:29]: Nit: maybe use single quotes `'GIMP'`, like the code around does.

File: eosclubhouse/quests/hack2/newcursor.py [hunk 5]
```diff
@@ -26,16 +26,13 @@ class NewCursor(Quest):
...
 
     def setup(self):
         self._app_terminal = App('org.gnome.Terminal')
         self._info_messages = self.get_loop_messages('NEWCURSOR')
 
     def step_begin(self):
-        if not self.app.is_installed():
-            self.wait_confirm('NOTINSTALLED', confirm_label='Got it!')
-            return self.step_abort
-
```
Comment (manuq) [inline eosclubhouse/quests/hack2/newcursor.py:38]: :+1:

File: eosclubhouse/quests/hack2/newcursor.py [hunk 6]
```diff
@@ -45,10 +42,7 @@ def step_begin(self):
 
     def step_launch(self):
         self.app.launch()
-        # self.app.launch(message_id='LAUNCHGLIMPSE')
         self.wait_confirm('LAUNCH')
-        # self.pause(4)
-        # self.wait_for_app_in_foreground(timeout=4)
```
Comment (manuq) [inline eosclubhouse/quests/hack2/newcursor.py:51]: :+1: to remove commented code.

**Review by manuq — CHANGES_REQUESTED** [review]
It has repeated lines. Other than that looks good!

**Comment by bkarnow** [comment]
Repeated lines removed, merging.

**Pull request merged**